src/c-writer.cc

```diff
@@ -1791,10 +1791,10 @@ void CWriter::WriteCallIndirectFuncDeclaration(const FuncDeclaration& decl,
 }
 
 void CWriter::WriteFeatureMacros() {
-  if (options_.features->exceptions_enabled()) {
+  if (module_->features.exceptions) {
```

Comment:
I thought the plan was to check both the command line switch and if the feature was in use?

Reply:
The command-line switches are enforced at read-time, so this can't get set unless it's enabled...

Reply:
ah gotcha